synapse/models/infotech.py

```diff
@@ -1921,22 +1936,13 @@ def getModelDefs(self):
                 )),
                 ('it:cmd', {}, ()),
                 ('it:exec:proc', {}, (
-                    ('host', ('it:host', {}), {
```

Comment:
The more descriptive doc strings may be useful enough to keep. Maybe we could make a way to declare them that inherits the type info from the interface but gets the more specific doc string. Lets do a quick goback :+1: